.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,3 +7,11 @@
 
 
 ### Specify the version of the product this PR applies to. 
+
+### Pull Request Checklist
+
+Make sure that we all those checks befor merging the PR:
+
+- [ ] `vale` validation
+- [ ] link checker validation
+- [ ] QE Team Approval
```

Comment:
Not sure if Che QE team will have resources to check each PR in here.

From this point of view, it might make sense to ask developers for approval as well, those who are familiar with functionality which changes are related.

Reply:
@dmytro-ndp @rhopp we want to make sure that what's in documentation is covered by QE test scenarios. That there is a test that verifies that what's described in the documentation works. The check can be performed by the author of the PR or the reviewer (with your support). If no scenario exists yet an issue for you should be created so that you add that in your list of things to tests. Does that work?